wicked/diagrams/operator_expression.cc

```diff
@@ -83,11 +85,30 @@ make_diag_operator_expression(const std::string &label,
         ann[space] += 1;
       }
     }
-    result.add({Operator(label, cre, ann)});
+    result.add({Operator(label, cre, ann)}, scalar_t(1, 1), unique);
   }
   return result;
 }
 
+OperatorExpression
```

Comment:
I would rewrite this in this way: define a single function with this signature:
```
make_diag_operator_expression(const std::string &label,
                              const std::vector<std::string> &components,
                              bool unique)
```
and then at the level of the python binding intruduce the logic to handle the two cases. This will keep the code cleaner. Ideally the C++ library should not be handling special cases, python should be responsible.

Reply:
Done!